orchestrator.py

```diff
@@ -37,14 +37,19 @@ def storing_and_sorting_items_data(db_queries, items_to_store):
     db_queries.add_prices(items_to_store)
     db_queries.check_for_lowest_price_and_update()
     id_list_to_pass = [(item[0],) for item in items_to_store]
-    logger.debug(f'{id_list_to_pass}')
+    logger.debug(f'id_list_to_pass: {id_list_to_pass}')
     target_price_list = db_queries.check_target_prices(tuple(id_list_to_pass))
```

Comment:
why are you converting it to tuple here?
also, should the name be something like `items_ids`?